test/integration/users_login_test.rb

```diff
@@ -25,8 +25,11 @@ def setup
 
   test "login with valid information followed by logout" do
     get login_path
-    post login_path, params: { session: { email: @user.email,
-                                          password: 'password' } }
+    post login_path, params: { session: 
+                              { email: @user.email,
+                                password: 'password' 
+                                } 
+                              }
```

Comment:
ditto extrating params into variable (see comment on previous PR and suggestion in next comment below)